src/client/wieldmesh.cpp

```diff
@@ -510,6 +524,22 @@ void getItemMesh(Client *client, const ItemStack &item, ItemMesh *result)
 	// Shading is on by default
 	result->needs_shading = true;
 
+	// Disable backface culling for the drawtypes below
+	bool cull_backface = true;
+	switch (f.drawtype) {
```

Comment:
This switch/case must be maintained in two places. Would you please be so nice and move this code into a `ContentFeatures` function (inline, perhaps?)